### PR TITLE
userid is cleaned by rstrip during password reset submission

### PR DESCRIPTION
**Pull request by nazrulworld**
There could be chance that user could copy his/her username with white space unintentionally. Thus I added a extra line for cleaning up.

### CODE REVIEW

File: Products/PasswordResetTool/skins/PasswordReset/pwreset_action.cpy [hunk 1]
```diff
@@ -12,6 +12,8 @@ from Products.PasswordResetTool.PasswordResetTool import InvalidRequestError, Ex
 status = "success"
 pw_tool = getToolByName(context, 'portal_password_reset')
 try:
+    # sanitization
+    userid = userid.rstrip()
```
Comment (mauritsvanrees) [inline Products/PasswordResetTool/skins/PasswordReset/pwreset_action.cpy]: I would use `strip`: white space in front of the userid would be weird too.
And I would keep this line out of the `try/except`.

Comment (nazrulworld) [reply Products/PasswordResetTool/skins/PasswordReset/pwreset_action.cpy]: @mauritsvanrees thanks for reply
I fully agree with you. I am doing that right now

**Comment by mauritsvanrees** [comment]
Can you still move the line above the try/except?  It is good form to let a try/except handle as few lines as possible. The exception that is caught cannot happen on your line, so it belongs outside the exception handling.

Also, please add an entry in `CHANGES.rst`.

And please start the tests on Jenkins:

- Log in to [jenkins.plone.org](https://jenkins.plone.org) with your github account.
- Go to the 'Build with parameters' form for all relevant Plone versions (only 5.0 here). You should see direct links to those versions under the 'Some checks haven’t completed yet' box of the pull request.
- Paste the pull request url in that form and build it.

**Comment by mauritsvanrees** [comment]
Seems good now. I will merge. Thanks!

**Pull request merged**